fem/transfer.hpp

```diff
@@ -207,6 +225,51 @@ class L2ProjectionGridTransfer : public GridTransfer
                          ElementTransformation* tr_lor,
                          IntegrationPointTransformation& ip_tr,
                          DenseMatrix& M_mixed_el) const;
+
+      void ElemMixedMass(Geometry::Type geom, const FiniteElement& fe_ho,
+                         const FiniteElement& fe_lor, ElementTransformation* el_tr,
+                         IntegrationPointTransformation& ip_tr,
+                         DenseMatrix& B_L, DenseMatrix& B_H) const;
+   public:
+      /*
+      Returns the Mixed Mass M_LH via device element assembly by building the basis functions and
+      data at the quadrature points.
+      */
+      Vector MixedMassEA(const FiniteElementSpace& fes_ho_,
+                         const FiniteElementSpace& fes_lor_,
+                         const Coefficient* coeff_, Vector &M_LH,
+                         MemoryType d_mt_ = MemoryType::HOST);
+   };
+
+   //Class below must be public as we now have device code
```

Comment:
Here and in other places, for better readability, please add a space after `//`.